services/paybuttonService.ts

```diff
@@ -154,7 +154,7 @@ export async function createPaybutton (values: CreatePaybuttonInput): Promise<Pa
   // This has to be done before, or the connection
   // on relation tables to these addresses will fail
   const addressIdList: string[] = []
-  const updatedAddressIdList: string[] = []
+  const updatedAddressStringList: string[] = []
   const createdAddresses: addressService.AddressWithTransactions[] = []
 
   // Prevent IFP addresses from being created
```

Comment:
Any reason to keep this (specifically referring to the IFP address, not blocking addresses in general)?

Reply:
I think this was implemented mixing the two concepts: IFP addresses are the ones we block. But this is a matter of variable naming (if I understood your question correctly), so would be only a matter of renaming the constant `IFP_ADRESSES` to `BLOCKED_ADDRESSES`.

Reply:
Ok let's do this then.